src/test/kotlin/day03/ClericTest.kt

```
@@ -0,0 +1,64 @@
+package day03
+
+import org.example.day03.Cleric
+import org.junit.Assert.*
+import kotlin.test.assertContains
+
+class ClericTest {
+
+    @org.junit.Before
+    fun setUp() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)